macros/EmbedInteractiveExample.ejs

```diff
@@ -0,0 +1,31 @@
+<%
+// Embeds a live sample from a mdn.github.io GitHub page
+//
+// Parameters:
+//  $0 - The URL of mdn.github.io page (relative)
+//  $1 - Keyword indicating iframe size: small, medium, or large
+//
+//  Example call {{EmbedInteractiveExample("interactive-examples/pages/css/animation.html", "small")}}
+//
+
+let url = "https://mdn.github.io/" + $0;
+let classesList = Array();
+
+// figure out classes
+if($0.includes('/js/')){
+    classesList.push('interactive-js');
```

Comment:
Thanks :). It's funny that this is different to the general standard for Mozilla code: https://developer.mozilla.org/en-US/docs/Mozilla/Developer_guide/Coding_Style#Indentation.